aptos-move/e2e-move-tests/src/tests/aggregator_v2.rs

```diff
@@ -382,6 +382,28 @@ proptest! {
         );
     }
 
+    #[test]
+    fn test_aggregator_with_republish(test_env in arb_test_env(6), element_type in arb_agg_type(), use_type in arb_use_type()) {
```

Comment:
Should somehow remember to adjust it (to failpoints) when we get rid of the fallback

Reply:
we should probably just have both?

Reply:
as after your PR , different path will be executed

what is the faillpoint name you have?

Reply:
good point

Reply:
fail-point-resource-group-serialization
https://github.com/aptos-labs/aptos-core/blob/e34cd18d6207c1422eaf6215638c93918b83445d/aptos-move/block-executor/src/executor.rs#L780C14-L780C53

it also enables in fallback though, but we can adjust it later.